theano/sandbox/cuda/cula.py

```diff
@@ -49,7 +49,12 @@ def make_thunk(self,
 
         # Initialize CULA the first time it is needed
         global cula_initialized
-        if cula_available and cula and not cula_initialized:
+
+        if not (cula_available and cula):
```

Comment:
I think the old condition wasn't good. I think it should be just:
```
if not cula_available:
```

cula won't be defined if it isn't available and it will be defined if it is available..  So this is a useless extra condition.